app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -438,7 +438,7 @@ type NimbusExperimentType {
   languages: [NimbusLanguageType!]!
   isSticky: Boolean
   projects: [NimbusProjectType]
-  hypothesis: String!
+  hypothesis: String
```

Comment:
What does this change do?

Reply:
previously we were not fetching the hypothesis in one of the graphql call, so that value as null was accepted. Since we need the hypothesis, this value can't be null-` String! # Can't return null`